internal/scaffold/cmd.go

```diff
@@ -192,16 +192,24 @@ func main() {
 // It serves those metrics on "http://metricsHost:operatorMetricsPort".
 func serveCRMetrics(cfg *rest.Config) error {
 	// Below function returns filtered operator/CustomResource specific GVKs.
+
+	// Get the namespace the operator is currently deployed in.
+	operatorNs, err := k8sutil.GetOperatorNamespace()
+    if err != nil {
+		// Check if is running out of the cluster
+		if err == k8sutil.ErrRunLocal {
```

Comment:
In go 1.13 we can do `errors.Is(err, k8sutil.ErrRunLocal)` to compare sentinel error values. Ditto elsewhere.

Reply:
cool ! tks for let me know either.